factories/receivers_aix.go

```diff
@@ -0,0 +1,128 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build aix
+
+package factories
+
+import (
```

Comment:
Has this list been audited to only include receivers that fit our immediate use-case? Some of them I can get behind but I cant help but feel that most of them are not something we need right away, and will be time expensive to test.

Reply:
It hasn't been fully vetted yet, since we have multiple customers talking about it. I'm waiting to further work on this part until one of them makes up their mind, and we discuss what they need.